src/Components/Common/Menu.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { Navbar, Nav, Button, NavDropdown, Form, FormControl } from 'react-bootstrap';
+import { Link } from 'react-router-dom';
+const Menu = () => {
+  return (
+    <Navbar bg="light" expand="lg">
+    <Navbar.Brand href="#home">React-Bootstrap</Navbar.Brand>
+    <Navbar.Toggle aria-controls="basic-navbar-nav" />
+    <Navbar.Collapse id="basic-navbar-nav">
+      <Nav className="mr-auto">
+        <Nav.Link as={Link} to="/user/sign_in">Sign In</Nav.Link>
```

Comment:
Why is sign in always showing? it should hide when the user is signed in already